src/ui/src/builder/journal/BuilderJournalEntryDetails.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+	<div class="BuilderJournalEntryDetails">
+		<!-- Entry Header -->
+		<div class="header">
+			<div class="header__title">
+				<h3>{{ entry.title }}</h3>
+				<WdsTag :text="entry.result" :variant="statusVariant" />
+			</div>
+			<div class="header__meta">
+				<span>{{ formattedDate }} {{ formattedTime }}</span>
+				<span>{{ entry.instanceTypeLabel }}</span>
+				<span>{{ entry.trigger.type }}</span>
+			</div>
+		</div>
+
+		<!-- Actions Bar -->
+		<div class="actions">
+			<WdsButton variant="neutral" size="small" @click="copyOutput">
+				<WdsIcon name="copy" />
+				Copy Output
+			</WdsButton>
+			<WdsButton variant="neutral" size="small" @click="reRunExecution">
+				<WdsIcon name="play" />
+				Re-run
+			</WdsButton>
+			<WdsButton variant="neutral" size="small" @click="goToTrigger">
+				<WdsIcon name="locate" />
+				Go to Trigger
+			</WdsButton>
+			<WdsButton variant="neutral" size="small" @click="downloadAsJson">
+				<WdsIcon name="download" />
+				Download JSON
+			</WdsButton>
+		</div>
+
+		<!-- Tabs for different sections -->
+		<div class="tabs">
+			<button
+				:class="{ active: activeTab === 'outputs' }"
+				@click="activeTab = 'outputs'"
+			>
+				Outputs
+			</button>
+			<button
+				:class="{ active: activeTab === 'metadata' }"
+				@click="activeTab = 'metadata'"
+			>
+				Metadata
+			</button>
+			<button
+				:class="{ active: activeTab === 'raw' }"
+				@click="activeTab = 'raw'"
+			>
+				Raw JSON
+			</button>
```

Comment:
`type=button` is needed; otherwise it's `type=submit` by default, and it'll submit any parent form.
```suggestion
			<button
				:class="{ active: activeTab === 'outputs' }"
				type="button"
				@click="activeTab = 'outputs'"
			>
				Outputs
			</button>
			<button
				:class="{ active: activeTab === 'metadata' }"
				type="button"
				@click="activeTab = 'metadata'"
			>
				Metadata
			</button>
			<button
				:class="{ active: activeTab === 'raw' }"
				type="button"
				@click="activeTab = 'raw'"
			>
				Raw JSON
			</button>
```

Also, did you look at `WdsTab` ?